src/NetMQ/OutgoingSocketExtensions.cs

```diff
@@ -98,8 +200,182 @@ public static IOutgoingSocket SendMore([NotNull] this IOutgoingSocket socket, [N
 
         #endregion
 
+        #endregion
+
+        #region Sending a multipart message as byte arrays
+
+        #region Blocking
+
+        public static void SendMultipartBytes([NotNull] this IOutgoingSocket socket, params byte[][] frames)
+        {
+            SendMultipartBytes(socket, (IEnumerable<byte[]>)frames);
+        }
+
+        public static void SendMultipartBytes([NotNull] this IOutgoingSocket socket, IEnumerable<byte[]> frames)
+        {
+            var enumerator = frames.GetEnumerator();
```

Comment:
Note sure it matters much, but I think .NET calls `Dispose` on any `IEnumerator<T>` which is `IDisposable`. Info on [MSDN](https://msdn.microsoft.com/en-us/library/aa664754(v=vs.71).aspx).